### DISCUSSION TITLE
默认助手重命名？

### DISCUSSION THREAD

**Discussion by RockStone** [lobehub/lobehub#1852] [General Question] [answered]
Hi folks, is it possible to rename the "Default Agent". all agents I create are named the same and I couldn't to find a way to rename them...

![image](https://github.com/lobehub/lobe-chat/assets/1649569/027fa078-4638-4452-818c-88ab1d454670)

**Comment by MapleEve** [comment, marked as answer]
<img width="381" alt="image" src="https://github.com/lobehub/lobe-chat/assets/1845053/0d54b733-9706-4940-be56-bd48404a09ac">
Click topright and there you go.

**Reply by arvinxx** [reply]
I think we can add an "edit" button to the `more` action icon

**Reply by MapleEve** [reply]
Yes, but currently the operations at this menu level can all be completed on the current page, while editing an Agent requires jumping to another page, disrupting the operation flow.
